lambdas/search/index.py

```diff
@@ -66,11 +66,14 @@ def lambda_handler(request):
     user_indexes = request.args.get('index', "")
     user_size = request.args.get('size', DEFAULT_SIZE)
     user_source = request.args.get('_source', [])
+    user_from = request.args.get('from')
     terminate_after = None  # see if we can skip os.getenv('MAX_DOCUMENTS_PER_SHARD')
 
     if not user_indexes or not isinstance(user_indexes, str):
         raise ValueError("Request must include index=<comma-separated string of indices>")
 
+    es_from = None
```

Comment:
you don't need this because variables declared inside the if statement are valid in the whole function

Reply:
done